tests/puppeteer/campaigns/functional/BO/catalog/products/filterAndQuickEditProducts.js

```diff
@@ -0,0 +1,212 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+const {Products} = require('@data/demo/products');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const ProductsPage = require('@pages/BO/catalog/products/index');
+
+let browser;
+let page;
+let numberOfProducts = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    productsPage: new ProductsPage(page),
+  };
+};
+
+// Filter Products
+describe('Filter Products', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO and go to products page
+  loginCommon.loginBO();
+
+  it('should go to "Catalog>products" page', async function () {
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.catalogParentLink,
+      this.pageObjects.boBasePage.productsLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.productsPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.productsPage.pageTitle);
+  });
+
+  it('should reset all filters and get Number of products in BO', async function () {
+    numberOfProducts = await this.pageObjects.productsPage.resetAndGetNumberOfLines();
+    await expect(numberOfProducts).to.be.above(0);
+  });
+  // 1 : Filter products with all inputs and selects in grid table
+  describe('Filter products', async () => {
+    it('should filter by Id MIN-MAX \'2-10\'', async function () {
+      await this.pageObjects.productsPage.filterIDProducts(2, 10);
+      const numberOfProductsAfterFilter = await this.pageObjects.productsPage.getNumberOfProductsFromList();
+      await expect(numberOfProductsAfterFilter).to.be.below(numberOfProducts);
+      /* eslint-disable no-await-in-loop */
```

Comment:
you can delete those `/* eslint-disable no-await-in-loop */` `/* eslint-enable no-await-in-loop */` in all tests (no need already disabled in eslintrc file)